ci-operator/step-registry/upi/deprovision/heterogeneous/powervs/upi-deprovision-heterogeneous-powervs-commands.sh

```diff
@@ -212,6 +227,18 @@ then
   cp "${PRIVATE_KEY_FILE}" "${IBMCLOUD_HOME_FOLDER}/ocp4-upi-compute-powervs/data/id_rsa"
   cp "${KUBECONFIG}" "${IBMCLOUD_HOME_FOLDER}/ocp4-upi-compute-powervs/data/kubeconfig"
 
+  #Create the VPC Connections for the TG
+  for GW in $(ic tg gateways --output json | jq --arg resource_group "${RESOURCE_GROUP_ID}" --arg workspace_name "${WORKSPACE_NAME}" -r '.[] | select(.resource_group.id == $resource_group) | select(.name == $workspace_name) | "(.id)"')
+  do
+    for CS in $(ic is vpcs --output json | jq -r '.[] | select(.name | contains("${WORKSPACE_NAME}-vpc")) | .id')
+    do
+      VPC_CONN_NAME=$(ic is vpc "${CS}" --output json | jq -r .name)
+      VPC_NW_ID=$(ic is vpc "${CS}" --output json | jq -r .crn)
+      echo "Creating new VPC connection for gateway now."
+      ic tg cc "${GW}" --name "${VPC_CONN_NAME}" --network-id "${VPC_NW_ID}" --network-type vpc || true
```

Comment:
Add a sleep after the loop finishes done/done... 
at this point you should also break.